api/cluster/controller.go

```diff
@@ -251,6 +251,7 @@ func (c *controller) Deploy(ctx context.Context, modelService *models.Service) (
 	}
 
 	if c.deploymentConfig.PodDisruptionBudget.Enabled {
+		// Create / update pdb
```

Comment:
instead of this maybe change `createPodDisruptionBudgets` to `generatePDBSpecs` ? since nothing is created until deployed